gcce.service.js

```diff
@@ -378,30 +378,33 @@ module.exports = class GoogleComputeService extends Compute {
     return (await iam.projects.serviceAccounts.list(request)).data.accounts;
   }
 
-  async listNetworks(_, fields, pageToken) {
+  async listNetworks({ query }, fields, pageToken) {
     const request = removeUndefinedAndEmpty({
       auth: this.getAuthClient(),
       project: this.projectId,
       fields: parseFields(fields),
+      filter: query && `name:${query}`,
```

Comment:
If query is falsy we're going to pass that falsy value there. Is that what you want?